chore: cleaned up readme

antsif-a opened the pull request:
### Overview
* renamed from .MD to .md
* corrected typos
* fixed markdown code style
* added more information & links
* removed HTML inline elements

antsif-a committed:
chore: cleaned up readme

* renamed from .MD to .md
* corrected typos
* fixed markdown code style
* added more information & links
* removed HTML inline elements

skykatik commented on README.md:
```diff
@@ -0,0 +1,25 @@
+* **Windows**: `gradlew dist`  
+* **Linux**: `./gradlew dist`
+
+After building, the .JAR file should be located in `build/libs/inside-bot.jar` folder.
```
_I forgot to rename my local repository._ Correct name is 'InsideBot.jar'

antsif-a committed:
fix: inside-bot -> InsideBot

https://github.com/MindustryINSIDE/InsideBot/pull/2#pullrequestreview-536114593
